.github/workflows/ci.yml

```diff
@@ -51,13 +51,13 @@ jobs:
       with:
         toolchain: stable
     - name: Run tests
-      run: cargo test --features serde
+      run: RUST_MIN_STACK=4194304 cargo test --features serde
```

Comment:
Alternatively to avoid duplication you can set env per step/job or for whole workflow.
see https://docs.github.com/en/actions/learn-github-actions/variables